next.config.js

```diff
@@ -1,5 +1,24 @@
 /** @type {import('next').NextConfig} */
 module.exports = {
   reactStrictMode: true,
-  target: 'serverless'
+  target: 'serverless',
+  redirects: async () => {
+    return [
+      {
+        source: '/academic-grants',
+        destination: '/applicants',
+        permanent: true
+      },
+      {
+        source: '/academic-grants/apply',
+        destination: '/applicants',
+        permanent: true
+      },
+      {
+        source: '/academic-grants/thank-you',
+        destination: '/applicants',
+        permanent: true
+      }
+    ];
+  }
```

Comment:
I've updated the `netlify.toml`. We need the redirects on the `next.config.js` to be able to see the changes locally though

Reply:
I think I'd rather use Netlify Dev to simulate changes in the local env because that more closely mimics production:
https://cli.netlify.com/netlify-dev/

@pettinarip curious what you think.